ci/scripts/install_python.sh

```diff
@@ -46,7 +47,12 @@ full_version=${versions[$2]}
 if [ $platform = "macOS" ]; then
     echo "Downloading Python installer..."
 
-    if [ "$(uname -m)" = "arm64" ] || [ "$version" = "3.10" ] || [ "$version" = "3.11" ] || [ "$version" = "3.12" ]; then
+    if [ "$(uname -m)" = "arm64" ] || \
+       [ "$version" = "3.10" ] || \
+       [ "$version" = "3.11" ] || \
+       [ "$version" = "3.12" ] || \
+       [ "$version" = "3.13" ];
+    then
         fname="python-${full_version}-macos11.pkg"
     else
         fname="python-${full_version}-macosx10.9.pkg"
```

Comment:
It seems the link is https://www.python.org/ftp/python/3.13.0/python-3.13.0rc1-macos11.pkg, so once the shorter version and once the full version (including "rc1").

With the current script it does try https://www.python.org/ftp/python/3.13.0rc1/python-3.13.0rc1-macos11.pkg